src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -654,17 +656,47 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("2.7");
 		}
+
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerStopUsingItemEvent")) {
+			Skript.registerEvent("Stop Using Item", SimpleEvent.class, PlayerStopUsingItemEvent.class,
+							"[player] (stop|end) (using item|item use)")
+					.description("Called when a player stops using an item. For example, when the player releases the " +
+							"interact button when holding a bow, an edible item, or a spyglass.",
+							"Note that event-number/event-timespan will return the number of ticks the item was held for.")
```

Comment:
I don't see a number event value being registered. Is this still accurate?

Reply:
ah thanks for the catch